packages/react-ui-kit/src/Menu/MenuModal.jsx

```diff
@@ -91,16 +91,20 @@ const MenuModalBackground = styled.div`
 
 const noop = () => {};
 
-const MenuModal = ({children, onBackgroundClick}) => (
-  <MenuModalWrapper>
+const MenuModal = ({children, onBackgroundClick, ...props}) => (
+  <MenuModalWrapper {...props}>
     <MenuModalBody>
       <MenuModalContent>{children}</MenuModalContent>
     </MenuModalBody>
     <MenuModalBackground onClick={onBackgroundClick} data-uie-name="menu-background" />
   </MenuModalWrapper>
 );
 
-const MenuItem = ({children, onClick}) => <MenuItemContent onClick={onClick}>{children}</MenuItemContent>;
+const MenuItem = ({children, onClick, ...props}) => (
+  <MenuItemContent onClick={onClick} {...props}>
```

Comment:
I believe you don't need to specify the `onClick={onClick}` since it can be handled by the `{...props}` spreading.